.checkov.yaml

```
@@ -0,0 +1,5 @@
+---
+skip-check:
+  - CKV2_ANSIBLE_3
```

Would it be useful/proportionate to include a comment describing *why* we disable checks? Something like:

```suggestion
  # Requires all blocks to have rescue: - not considered appropriate
  - CKV2_ANSIBLE_3
```